Remove Character From String/RemoveCharacterFromString.js

```diff
@@ -0,0 +1,7 @@
+// *Challenge 4
+// Write a JavaScript program to remove a character at the specified position of a given string and return the new string.
+
+// console.log(removeCharacter('Example', 0)) => should log xample
+
+let sentence = "This is a short sentence.";
```

Comment:
The task was to create a JavaScript program _function_ that expects two parameters and returns a new string. The first parameter should be a string and the second parameter should be the index of the removed letter.

When executed the function should return the new string with the removed character from it _e.g._ `removeCharacter('Example', 2) `  - - -returns `'Exmple'`